definitions/npm/redux-act_v1.x.x/flow_v0.34.x-/redux-act_v1.x.x.js

```diff
@@ -0,0 +1,49 @@
+declare module 'redux-act' {
+  declare module.exports: {
+    createReducer<ReducerState>(
+      handlers: {
+        [key: string]: (ReducerState, payload: mixed) => ReducerState,
+      },
```

Comment:
handlers may be Object or Function.
https://github.com/pauldijou/redux-act/blob/master/src/createReducer.js#L49